src/main/java/net/greatstart/controllers/UserRestController.java

```diff
@@ -0,0 +1,74 @@
+package net.greatstart.controllers;
+
+
+import net.greatstart.dto.DtoUserProfile;
+import net.greatstart.mappers.UserProfileMapper;
+import net.greatstart.model.User;
+import net.greatstart.services.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+
+import javax.validation.Valid;
+import java.security.Principal;
+
+@RestController
+public class UserRestController {
```

Comment:
Word Rest is not needed in controller name. All your controller will be rest ones